src/ScenarioPreview/ScenarioPreview.tsx

```diff
@@ -0,0 +1,152 @@
+import FormatVersionLabel from "../FormatVersionLabel/FormatVersionLabel";
+import NoWrapTypography from "../NoWrapTypography/NoWrapTypography";
+import React from "react";
+import RoadLabel from "../RoadLabel/RoadLabel";
+import { ScenarioPreviewProps } from "./ScenarioPreview.types";
+import UserLabel from "../UserLabel/UserLabel";
+
+export function ScenarioPreview({
+  name,
+  href,
+  image,
+  description,
+  format,
+  formatVersion,
+  file,
+  createdAt,
+  user,
+  label = [],
+  roadLabel,
+  sx
+}: ScenarioPreviewProps) {
+  function hasOptionalProperty() {
+    return label?.length > 0 || createdAt || user;
+  }
+
+  const labelChips: LabelChipGroupProps["chips"] = label?.map(l => ({
+    clickable: false,
+    color: l.color,
+    label: l.name,
+    size: "small"
+  }));
+
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      gap={1}
+      minWidth={0}
+      fontFamily="Montserrat"
+      data-testid="scenario-preview-wrapper"
+      sx={{ ...sx }}
+    >
+      <Box>
+        <Stack direction="row" spacing={1} minWidth={0}>
+          <img
+            src={image}
+            alt="scenario-image"
+            style={{
+              height: "44px",
+              marginBottom: "auto",
+              marginTop: "auto",
+              objectFit: "contain",
+              width: "78px"
+            }}
+          />
+          <Stack direction="column" minWidth={0}>
+            <NoWrapTypography>
+              <Link
+                href={href}
+                target="_blank"
+                color="primary"
+                variant="subtitle2"
+                underline="hover"
+                data-testid="scenario-preview-name"
+              >
+                {name}
+              </Link>
+            </NoWrapTypography>
+
+            <Typography
+              color="textPrimary"
+              data-testid="scenario-preview-description"
+              sx={{
+                WebkitBoxOrient: "vertical",
+                WebkitLineClamp: 2,
+                display: "-webkit-box",
+                overflow: "hidden",
+                textOverflow: "ellipsis"
+              }}
+              variant="caption"
+            >
+              {description}
+            </Typography>
+          </Stack>
+        </Stack>
+      </Box>
+      <Box>
+        <Stack
+          direction="row"
+          gap={1}
+          justifyContent="left"
+          alignItems="center"
+        >
+          <Box data-testid="format-label" flex="0 1 auto">
+            <FormatLabel label={format} />
+          </Box>
+          <Box data-testid="format-version-label" flex="0 1 auto">
+            <FormatVersionLabel label={formatVersion} />
+          </Box>
+          <Box
+            data-testid="file-label"
+            flex="0 1 auto"
+            maxWidth={"calc(40% - 12px)"}
+            alignItems="center"
+          >
+            <FileLabel label={file} />
+          </Box>
+          {roadLabel && (
```

Comment:
`roadLabel` is not an optional param, the `RoadLabel` component should always be displayed.